tensorboard/plugins/scalar/scalars_plugin.py

```diff
@@ -85,11 +125,36 @@ def index_impl(self):
 
   def scalars_impl(self, tag, run, output_format):
     """Result of the form `(body, mime_type)`."""
-    tensor_events = self._multiplexer.Tensors(run, tag)
-    values = [[tensor_event.wall_time,
-               tensor_event.step,
-               tf.make_ndarray(tensor_event.tensor_proto).item()]
-              for tensor_event in tensor_events]
+    if self._db_connection_provider:
+      db = self._db_connection_provider()
+      cursor = db.execute('''
+        SELECT
+          Tensors.step,
+          Tensors.computed_time,
+          Tensors.data,
+          Tensors.dtype,
+        FROM Tensors
+        LEFT JOIN Tags
+        ON Tensors.series=Tags.tag_id
```

Comment:
Indent +2 space here. Space around equals please.

Perfecting style minimizes distractions.

Also, please consider lending five more minutes to reading rhetoric on go/readability page.

Then guess whose prose 'twas.

Reply:
Thanks for the catch. Nice, articulate prose on readability. :)